src/ecrc-frontend/src/components/page/error/Error.js

```diff
@@ -48,7 +71,9 @@ export default function Error({ page: { header, error } }) {
           again later.
           <br />
           <br />
-          {error}
+          {error.status}
```

Comment:
i dont think we should show the status 
it is not useful to end user

Reply:
removed